static/js/tools/statsvar_menu.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from "react";
 import hierarchy from "../../../tools/pv_tree_generator/hierarchy.json";
+const menujson=[hierarchy[0]];
```

Comment:
can just make the entire hierarchy a dictionary instead of a list, since now there is only one element

Reply:
Done.